blackjax/mcmc/trajectory.py

```diff
@@ -617,3 +618,18 @@ def update_sum_log_p_accept(inputs):
         return expansion_state, (is_diverging, is_turning)
 
     return expand
+
+
+def stochastic_trajectory(
```

Comment:
I feel like `trajectory.py` can be left as is, it's just a matter of building an `IntegratorState` from `rmh.py` and passing it to the `proposal_generator`.

Reply:
Done